modules/features/falcon_feature_commerce/config/install/config_snapshot.snapshot.config_sync.module.commerce_directdebit.yml

```diff
@@ -0,0 +1,10 @@
+langcode: en
```

Comment:
We shouldn't be including config_snapshot configs into features - it is development-only module